### DISCUSSION TITLE
Draining the queue

### DISCUSSION THREAD

**Discussion by latinvm** [swar/Swar-Chia-Plot-Manager#203] [Ideas]
I have just installed this plot manager yesterday and really like how it makes CLI-plotting a lot easier to manage.

I was wondering if there is a setting that effectively allows you to "drain the queue", say you want to restart your machine and have a couple of plots nearing completion. You want to finish those plots but don't want to start any new plots. A switch that effectively disables any new plots to be started would be very useful.

Anyway, thanks for the useful tool :)

**Comment by jduraes** [comment]
If I understood correctly, all you need is to stop the Manager. Existing plots will continue, no new ones will be added to the queue, and you can still use the Viewer to monitor ongoing plots.

**Reply by latinvm** [reply]
Ah, good to know you are still able to monitor ongoing plots after you have stopped the manager!
